### ISSUE TITLE
ENH: Improve for Supersonic Aerodynamics

### ISSUE THREAD

**Issue by giovaniceotto** [RocketPy-Team/RocketPy#272]
RocketPy implements Barrowman's equations for subsonic compressible flow. As more and more flights achieve the supersonic regime, it is crucial that our aerodynamic model stays accurate at such conditions.

Therefore, this issue requests the implementation of Barrowman's equations considering supersonic flow.

## Requirements

1. Have a reasonably accurate model of normal aerodynamic forces for subsonic, transonic and supersonic flight.

2. Be able to evaluate the stability margin as a function of Mach number.

## Proposed Milestone

**v1.1.0**

**Comment by robertobilbao** [comment]
Is there any method to simulate the trayectory of supersonic Rockets on RocketPy?

**Comment by Gui-FernandesBR** [comment]
Regarding the 2 required features of this issue:

> Have a reasonably accurate model of normal aerodynamic forces for subsonic, transonic and supersonic flight.

We recently released the GenericSurfaces, which allowed for the import of completly customized aero curves! https://github.com/RocketPy-Team/RocketPy/pull/680

> Be able to evaluate the stability margin as a function of Mach number.

This was solved one year ago!
![image](https://github.com/user-attachments/assets/c09ae37f-aa11-4721-9902-385d0d890817)


With that being said, I'm marking this issue as solved.
